src/ISC.xml

```diff
@@ -6,7 +6,7 @@
          <crossRef>http://www.opensource.org/licenses/ISC</crossRef>
       </crossRefs>
       <titleText>
-         <p>ISC License:</p>
+         <p><alt match="The " name="titleThe"></alt>ISC License<alt match=" \(ISC[L]{0,1}\)" name="titleID"></alt><alt match=":" name="titleColon"></alt></p>
```

Comment:
Here's an alternative alt tag that uses a single alt which should improve performance on the matching:

`<alt match="(The )?ISC License( \(ISC\))?:?"></alt>`

Reply:
I tested the recommended match with the following:

ISC License:
ISC License
The ISC License
The ISC License (ISC)
The ISC License (ISC):